cfgov/unprocessed/js/organisms/FeaturedContentModule.js

```diff
@@ -0,0 +1,69 @@
+/* ==========================================================================
+   Featured Content Module Class
+   ========================================================================== */
+
+// Required modules.
+import { checkDom, setInitFlag } from '../modules/util/atomic-helpers';
+import VideoPlayer from './VideoPlayer';
+
+const BASE_CLASS = 'o-featured-content-module';
+
+let _dom;
+let _videoPlayer;
+
+/**
+ * FeaturedContentModule
+ * @class
+ *
+ * @classdesc Initializes a new FeaturedContentModule organism.
+ *
+ * @param {HTMLNode} element
+ *   The DOM element within which to search for the organism.
+ * @returns {FeaturedContentModule} An instance.
+ */
+function FeaturedContentModule( element ) {
+  _dom = checkDom( element, BASE_CLASS );
+
+  this.init = init;
+
+  return this;
+}
+
+/**
+ * @returns {FeaturedContentModule} An instance.
+ */
+function init() {
+  setInitFlag( _dom );
+
+  const videoPlayerDom = _dom.querySelector( `.${ VideoPlayer.BASE_CLASS }` );
+
+  // If we don't have a video in this FCM, bail out.
+  if ( videoPlayerDom === null ) {
+    return this;
+  }
```

Comment:
This is helpful, but my point was that we shouldn't even request and download this JS. Still think it's out of scope for this PR, though.